public/app/plugins/datasource/loki/metric_find_query.test.ts

```diff
@@ -0,0 +1,45 @@
+import { processMetricFindQuery } from './metric_find_query';
+import { makeMockLokiDatasource } from './mocks';
+import LokiDatasource from './datasource';
+
+const mocks = makeMocks();
+
+describe('processMetricFindQuery', () => {
+  mocks.forEach((mock, index) => {
```

Comment:
Nitpick: there is an `it.each` that I think you could use here. Just learned about it last week.

Reply:
Thanks Hugo, I'll check it out!